esphome/components/uart/uart_component_esp32_arduino.cpp

```diff
@@ -109,6 +109,11 @@ void ESP32ArduinoUARTComponent::setup() {
     this->number_ = next_uart_num;
     this->hw_serial_ = new HardwareSerial(next_uart_num++);  // NOLINT(cppcoreguidelines-owning-memory)
   }
+
+  load_settings(false);
```

Comment:
Please ensure that you are prefixing member variables and function calls
```suggestion
  this->load_settings(false);
```